README.md

```diff
@@ -0,0 +1,16 @@
+# star_movie
```

Comment:
due to the fact that you made the file pubspeck.yaml in the master branch, I can’t leave comments on some points of this PR, namely:

- pubspec.yaml: remove domain;
- pubspec.yaml: remove get_it;